pkg/workload/tpcc/tpcc.go

```diff
@@ -260,7 +262,8 @@ var tpccMeta = workload.Meta{
 			`deprecated-fk-indexes`:    {RuntimeOnly: true},
 			`query-trace-file`:         {RuntimeOnly: true},
 			`fake-time`:                {RuntimeOnly: true},
-			"txn-preamble-file":        {RuntimeOnly: true},
+			`txn-preamble-file`:        {RuntimeOnly: true},
+			`as-of`:                    {RuntimeOnly: true, CheckConsistencyOnly: true},
```

Comment:
Nit: it's typically abbreviated as `aost`. Most folks are familiar with it, but not `as-of`.